src/index.js

```diff
@@ -73,6 +75,46 @@ const renderFragment = async (path) => {
   };
 };
 
+/**
+ * Renders pages from AEM
+ */
+const renderAEMPages = async (path, params) => {
+  const { authorization, wcmmode } = params;
+  const aemURL = `${aemConfig.aemEnv}${path}.html`;
```

Comment:
Append wcmmode=disabled to the AEM URL to get rid of cq component decoration tags

Reply:
Fixed